azure-stack/hci/concepts/host-network-requirements.md

```diff
@@ -44,7 +44,7 @@ For more information about this role-based NIC qualification, please see this [l
  
 > [!NOTE]
 > The highest qualification for any adapter in our ecosystem will contain the **Management**, **Compute Premium**, and **Storage Premium** qualifications.
- 
+
 ![image](https://user-images.githubusercontent.com/12801954/188225569-bb160be0-96a2-4563-97d5-3d8efb3cb597.png)
```

Comment:
```suggestion
```
images must be hosted in the repo. @LouisBerner - I realize that your edits did not change this line, but it must be removed (and the ALT text needs to be much more descriptive).